pkg/cmd/build/depot_test.go

```diff
@@ -45,41 +45,6 @@ func (m *fakeDepotMachine) Connect(ctx context.Context) (*buildkitClient.Client,
 	return nil, m.err
 }
 
-func Test_isDepotEnabled(t *testing.T) {
```

Comment:
why removing the test instead of updating it?

Reply:
I don't think this test added any value the way it was tbh. Now `IsDepotEnabled` depends on two envvars and adding a test for that seems unnecessary.

Reply:
Nit: the test is not only to verify the current implementation (which I agree is trivial), but also to make sure that next changes in the function don't brake the functionality. 

I could add a change in the function that breaks it without realizing.

Anyways, no strong opinion on keeping or removing the function, as this is a very simple case. It was just a note